src/mbgl/renderer/render_orchestrator.cpp

```diff
@@ -141,6 +142,10 @@ std::unique_ptr<RenderTree> RenderOrchestrator::createRenderTree(
     if (!isMapModeContinuous) {
         // Reset zoom history state.
         zoomHistory.first = true;
+        if (stillImageRequest != updateParameters->stillImageRequest) {
+            clearData();
```

Comment:
This breaks caching for still mode. Snapshotter, render tests.

Reply:
thanks! than let's make it a map option